core/templates/domain/exploration/read-only-exploration-backend-api.service.ts

```diff
@@ -237,6 +241,31 @@ export class ReadOnlyExplorationBackendApiService {
     this._explorationCache = {};
   }
 
+  /**
+   * Retrives the Progress in a chapters
```

Comment:
```suggestion
   * Retrives the progress in chapters.
```

Apologies, I made a typo earlier. Drop the "a".